### PR TITLE
Update to tb-profiler 2.6.1

### PR DESCRIPTION
**Pull request by pvanheus**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by pvanheus** [comment]
This PR does a few things:

1. It updates the dependency to version 2.6.1
2. It adds the `--call_whole_genome` advanced option.

2.6.1 and the associated conda package tb-profiler-2.6.1--py_2 fixes issues that made `tb-profiler` break when running from a Singularity container. It also provides options that should allow for `vcf_profile` command to be added in the future, although this update does not add that command.

**Comment by bgruening** [comment]
@pvanheus is that needed in your workshop? When do you need it?

File: tools/tb-profiler/tb_profiler_profile.xml [hunk 1]
```diff
@@ -173,7 +176,7 @@
 Summary
 =======
 
-The pipeline aligns reads to the H37Rv reference using bowtie2, BWA or minimap2 and then calls variants using SAMtools. These variants are then compared to a drug-resistance database. We also predict the number of reads supporting drug resistance variants as an insight into hetero-resistance (not applicable for minION data).
+The pipeline aligns reads to the H37Rv reference using bowtie2, BWA or minimap2 and then calls variants using GATK. These variants are then compared to a drug-resistance database. We also predict the number of reads supporting drug resistance variants as an insight into hetero-resistance (not applicable for minION data).
```
Comment (bgruening) [inline tools/tb-profiler/tb_profiler_profile.xml]: we do not offer GATK as of now ...

Comment (pvanheus) [reply tools/tb-profiler/tb_profiler_profile.xml]: > we do not offer GATK as of now ...

GATK4 is bundled together with the `tb-profiler` conda recipe, so there is no need to add GATK as a tool

Comment (pvanheus) [reply tools/tb-profiler/tb_profiler_profile.xml]: I noticed that `tb-profiler` on usegalaxy.eu is broken at the moment and was planning to skip demonstrating it. If you have time to update immediately after merge though, I could demonstrate the tool (on Wednesday). If you don't have time, don't worry though - I can demonstrate on SANBI Galaxy and tell people to wait a while for usegalaxy.eu to update.

**Comment by bgruening** [comment]
I will trigger a reinstall later in the evening ...

**Comment by bgruening** [comment]
Should be installed ...

**Pull request merged**